### PR TITLE
Add splash screen, preload images, use native driver

### PR DESCRIPTION
**Pull request by brentvatne**

### CODE REVIEW

File: app/scenes/Schedule/index.js [hunk 1]
```diff
@@ -40,6 +39,7 @@ type Props = {
 };
 
 type State = {
+  hasScrolled: boolean,
```
Comment (JedWatson) [inline app/scenes/Schedule/index.js:42]: what is this being used for? I couldn't see an obvious purpose for tracking it

File: app/scenes/Schedule/index.js [hunk 2]
```diff
@@ -241,14 +249,17 @@ export default class Schedule extends Component {
         {/* Spacer for the headings to stick correctly */}
         <View style={styles.spacer} />
 
-        <ListView
+        <AnimatedListView
```
Comment (JedWatson) [inline app/scenes/Schedule/index.js:252]: Not sure why this is Animated?

Comment (brentvatne) [reply app/scenes/Schedule/index.js:252]: because it uses a native animated event, it needs to be

File: app/scenes/Schedule/index.js (hunk 1 above)
Comment (brentvatne) [reply app/scenes/Schedule/index.js:42]: unfortunately there is a bug in native scroll driven animations that hasn't been resolved in 0.41 (which exponent is currently using) and it causes the values to not be properly initialized, so I used this as a work around

**Pull request merged**